object-group-game/Database/WorldModel.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace object_group_game.Database
+{
+	public class Item
+	{
+		public int ID { get; set; }
+		public string DisplayName { get; set; }
+	}
+
+	public class Location
+	{
+		public int ID { get; set; }
+		public string DisplayName { get; set; }
+		public LocationType Type { get; set; }
+		public double Latitude { get; set; }
+		public double Longtitude { get; set; }
+		public int Radius { get; set; }
+		public int ItemID { get; set; }
+	}
+
+	public class ItemEffects
+	{
+		public int ItemID { get; set; }
+		public int EffectID { get; set; }
+		public int Value { get; set; }
+	}
+}
```

Comment:
This class shouldn't exist at all. It looks as if you are creating the same classes two times, and you should be able to use previous classes to do everything DB-related.

Reply:
I did not work with Entity Framework yet, but I did consult another team about the approach. And I can assure you, that it is possible to do this without creating a separate class for fields. That is the Entity Frameworks main goal. https://docs.microsoft.com/en-us/dotnet/framework/data/adonet/ef/overview